src/node/install.sh

```diff
@@ -309,7 +309,7 @@ curl -so- "https://raw.githubusercontent.com/nvm-sh/nvm/v${NVM_VERSION}/install.
     curl -so- "https://raw.githubusercontent.com/nvm-sh/nvm/\${PREV_NVM_VERSION}/install.sh" | bash
     NVM_VERSION="\${PREV_NVM_VERSION}"
 }
-source "${NVM_DIR}/nvm.sh"
+[ -s "\$NVM_DIR/nvm.sh" ] && . "\$NVM_DIR/nvm.sh"
```

Comment:
@samruddhikhandale

> @gvatsal60 Curious, why do we want to escape the $ sign?
  => As NVM_DIR is already set, it can be escaped here. However, to maintain consistency, I followed the suggestion.

> Can we continue using source instead of .?
    => Yes, that’s a great choice! Since this is a Bash script and not POSIX compliant, using `source` works well.